spec/models/merchant_spec.rb

```diff
@@ -1,8 +1,6 @@
 require 'rails_helper'
 
 RSpec.describe Merchant, type: :model do
-  describe "relationships" do
-    it {should have_many(:items)}
-    it {should have_many(:invoices)}
-  end
+  it { should have_many(:invoices) }
```

Comment:
Why'd we get rid of the describe relationships block here? I think we should leave it for clarity